src/post/schemas/post.schema.ts

```diff
@@ -23,6 +23,7 @@ export class Post {
 
     @Prop()
     likes: [];
+  id: any;
```

Comment:
delete this Prop, we dont need declare this prop. read this (https://stackoverflow.com/questions/71190267/nest-js-how-to-correctly-set-id-property-in-your-schema) to know more

Reply:
I have edited according to your feedback